public/index.html

```diff
@@ -32,23 +38,30 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>GT Scheduler</title>
-    <meta name="description"
-          content="Georgia Tech Scheduler lets you find the schedule that fits you best among all the possible combinations of courses.">
-    <meta name="author" content="Jinseo Jason Park">
-    <meta property="og:image" content="https://jasonpark.me/gt-scheduler/screenshot.png"/>
-    <meta property="og:type" content="website"/>
-    <meta property="og:title" content="GT Scheduler"/>
-    <meta property="og:url" content="https://jasonpark.me/gt-scheduler"/>
-    <meta property="og:description"
-          content="Georgia Tech Scheduler lets you find the schedule that fits you best among all the possible combinations of courses."/>
-</head>
-<body>
-<noscript>
-    You need to enable JavaScript to run this app.
-</noscript>
-<div id="root"></div>
-<!--
+    <title>GT Scheduler V2</title>
+    <meta
+      name="description"
+      content="Georgia Tech Scheduler lets you find the schedule that fits you best among all the possible combinations of courses."
+    />
+    <meta name="author" content="" />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/abhitirumala/gt-scheduler-v2/master/public/image.jpg"
+    />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="GT Scheduler V2" />
+    <meta property="og:url" content="https://gtscheduler.tech" />
+    <meta
+      property="og:description"
+      content="Georgia Tech Scheduler V2 lets you find the schedule that fits you best among all the possible combinations of courses."
+    />
+  </head>
+  <body>
+    <noscript>
+      You need to enable JavaScript to run this app.
+    </noscript>
+    <div id="root"></div>
+    <!--
```

Comment:
Don't change `title`, `og:image`, `og:title`, and `og:url`. For `author`, you can change to `Jinseo Park and [Your Name]`.